### DISCUSSION TITLE
make gui_6_final.def, gui_6_final.def and klayout_6_final.odb

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD-flow-scripts#1021] [General] [answered]
Is there some documentation on the targets above do and how they can be used?

I know that is a pretty open ended question, I'm just wondering what they are and if they could be useful for me.

**Comment by maliberty** [comment]
They load you open the .odb from any step in the flow.  The klayout targets are from before we had a GUI on OR so I rarely use them anymore except on the final gds.

**Reply by oharboe** [reply]
What is the difference between gui..odb and gui...def?

**Reply by maliberty** [reply, marked as answer]
..def is from when we didn't use .odb in ORFS.   You can still generate a def for any step but it isn't very useful in usual usage.
